src/Umbraco.Web/Editors/UserGroupsController.cs

```diff
@@ -87,5 +86,19 @@ public UserGroupDisplay GetUserGroup(int id)
 
             return display;
         }
+
+        public HttpResponseMessage PostDeleteUserGroups([FromUri] int[] userGroupIds)
```

Comment:
Only thing here is that for all delete methods by our conventions we allow both DELETE and POST verbs for delete actions, i'll update.